### PR TITLE
refactor(sandbox): replaced static auth util with injectable service

### PR DESCRIPTION
**Pull request by krsteen**

### CODE REVIEW

File: src/Sepes.Infrastructure/Service/DataModelService/SandboxModelService.cs [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+            if (sandbox == null)
+            {
+                throw NotFoundException.CreateForEntity("Sandbox", sandboxId);
+            }
+
+            await CheckAccesAndThrowIfMissing(sandbox, operation);
+
+            return sandbox;
+        }
```
Comment (HansKristianVikOlsen) [inline src/Sepes.Infrastructure/Service/DataModelService/SandboxModelService.cs:62]: Looks clean! Not much logic so little to look at

File: src/Sepes.Infrastructure/Service/DataModelService/SandboxModelService.cs [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+        public async Task<Sandbox> GetByIdWithoutPermissionCheckAsync(int sandboxId)
+        {
+            return await SandboxBaseQueries.AllSandboxesBaseQueryable(_db).SingleOrDefaultAsync(sb => sb.Id == sandboxId);
+        }
+
+        public async Task<Sandbox> GetByIdAsync(int id, UserOperation userOperation, bool withIncludes = false, bool disableTracking = false)
+        {
+            var sandboxQueryable =
+               (withIncludes ? SandboxBaseQueries.ActiveSandboxesWithIncludesQueryable(_db) : SandboxBaseQueries.ActiveSandboxesMinimalIncludesQueryable(_db));
```
Comment (HansKristianVikOlsen) [inline src/Sepes.Infrastructure/Service/DataModelService/SandboxModelService.cs:32]: Is this an SQL statement?

**Review by HansKristianVikOlsen — APPROVED** [review]
Looks good! I had one question

File: src/Sepes.Infrastructure/Service/DataModelService/SandboxModelService.cs (hunk 2 above)
Comment (krsteen) [reply src/Sepes.Infrastructure/Service/DataModelService/SandboxModelService.cs:32]: Not in its self, but inside SandboxBaseQueries lies linq queries related to sandbox

**Pull request merged**